lib/plugins/physics.js

```diff
@@ -192,6 +192,43 @@ function inject (bot, { physicsEnabled, maxCatchupTicks }) {
 
   bot.physics = physics
 
+  function getEffectLevel (mcData, effectName, effects) {
+    const effectDescriptor = mcData.effectsByName[effectName]
+    if (!effectDescriptor) {
+      return 0
+    }
+    const effectInfo = effects[effectDescriptor.id]
+    if (!effectInfo) {
+      return 0
+    }
+    return effectInfo.amplifier + 1
+  }
+
+  bot.elytraFly = async () => {
+    if (bot.entity.elytraFlying) {
+      throw new Error('Already elytra flying')
+    } else if (bot.entity.onGround) {
+      throw new Error('Unable to fly from ground')
+    } else if (bot.entity.isInWater) {
+      throw new Error('Unable to elytra fly while in water')
+    }
+
+    const mcData = require('minecraft-data')(bot.version)
+    if (getEffectLevel(mcData, 'Levitation', bot.entity.effects) > 0) {
+      throw new Error('Unable to elytra fly with levitation effect')
+    }
+
+    const item = bot.inventory.slots[6]
```

Comment:
This value should probably not be hardcoded

Reply:
Changed to getEquipmentDestSlot